release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -274,10 +273,86 @@ deprecated in {product-title} 3.11 and has been removed. You must use the
 deprecated in {product-title} 3.11 and have been removed. You must use the
 `--suggest` flag with this command instead.
 
+[id="ocp-4-5-service-catalog-removed"]
```

Comment:
Do you also want to add `REM` to table 1 of "Deprecated and removed features"?

Reply:
I think we're doing `-` for removed? Already had that there so I didn't futz with it.

Reply:
`REM` would be more clear tho probably. `-` could be interpreted as "Not applicable" (even tho the legend is right above...).

Reply:
You're correct, I glanced too quickly, bypassed the legend, and forgot we use `-` for removed. Just saw DEP and thought REM should be the other alternative. I agree REM would be more clear but I hate to drag @codyhoag back into this worm can again. :) Unless he agrees.

Reply:
@adellape @bobfuru good point. Since `-` is already used in the TP table for **Not Available**, it would probably be clearer to use something different for **Removed**. I think `REM` sounds good. @adellape do you mind including that update in this PR?

Reply:
I will do this!